planner-api/src/test/java/pl/edu/agh/kuce/planner/event/integration/OneTimeEventTest.java

```diff
@@ -57,7 +57,7 @@ void multiple_one_time_events_are_saved_properly_in_database()
                 new Timestamp(System.currentTimeMillis()));
         oneTimeEventRepository.save(testEvent1);
         oneTimeEventRepository.save(testEvent2);
-        Collection<OneTimeEvent> result = oneTimeEventRepository.findByUsersNick("TEST");
+        Collection<OneTimeEvent> result = oneTimeEventRepository.findByUserId(user.getId());
         assertThat(result.toArray()[0]).isEqualTo(testEvent1);
```

Comment:
Perhaps instead of calling result.toArray() again and again, findByUserId/findByUser could return List<> instead of Collection<>? If you expect data to be returned from database in certain order, it'd be better to return List<>, because List<> maintains insertion order and has get(index) method. Collection<> is a top level interface and we cannot assume that Hibernate won't use an implementation like HashSet<> (which doesn't maintain order) in the future. That could break the test.